content/en/docs/concepts/workloads/pods/pod-qos.md

```diff
@@ -0,0 +1,117 @@
+Pods that are `Burstable` have some lower-bound resource guarantees based on the request, but
+do not require a specific limit. If a limit is not specified, it defaults to a
+limit equivalent to the capacity of the Node, which allows the Pods to flexibly increase
+their resources if resources are available. In the event of Pod eviction due to Node
+resource pressure, these Pods are evicted only after all `BestEffort` Pods are evicted.
+Because a `Burstable` Pod can include a Container that has no resource limits or requests, a Pod
+that is `Burstable` can try to use any amount of node resources.
+
+#### Criteria
+
+A Pod is given a QoS class of `Burstable` if:
+
+* The Pod does not meet the criteria for QoS class `Guaranteed`.
+* At least one Container in the Pod has a memory or CPU request or limit.
+
+### BestEffort
+
+Pods in the `BestEffort` QoS class can use node resources that aren't specifically assigned
+to Pods in other QoS classes. For example, if you have a node with 16 CPU cores available to the
+kubelet, and you assign assign 4 CPU cores to a `Guaranteed` Pod, then a Pod in the `BestEffort`
+QoS class can try to use any amount of the remaining 12 CPU cores.
+
+The kubelet prefers to evict `BestEffort` Pods if the node comes under resource pressure.
+
+#### Criteria
+
+A Pod has a QoS class of `BestEffort` if it doesn't meet the criteria for either `Guaranteed`
+or `Burstable`. In other words, a Pod is `BestEffort` only if none of the Containers in the Pod have a
+memory limit or a memory request, and none of the Containers in the Pod have a
+CPU limit or a CPU request.
+Containers in a Pod can request other resources (not CPU or memory) and still be classified as
+`BestEffort`.
+
+### Some behavior is independent of QoS class
+
+Certain behavior is independent of the QoS class assigned by Kubernetes. For example:
+
+* Any Container exceeding a resource limit will be killed and restarted by the kubelet without
+  affecting other Containers in that Pod.
+
+* If a Container exceeds its resource request and the node it runs on faces
+  resource pressure, the Pod it is in becomes a candidate for [eviction](/docs/concepts/scheduling-eviction/node-pressure-eviction/).
+  If this occurs, all Containers in the Pod will be terminated. Kubernetes may create a
+  replacement Pod, usually on a different node.
+
+* The resource request of a Pod is equal to the sum of the resource requests of
+  its component Containers, and the resource limit of a Pod is equal to the sum of
+  the resource limits of its component Containers.
+
+* The kube-scheduler does not consider QoS class when selecting which Pods to
+  [preempt](/docs/concepts/scheduling-eviction/pod-priority-preemption/#preemption).
+  Preemption can occur when a cluster does not have enough resources to run all the Pods
+  you defined.
+
+## {{% heading "whatsnext" %}}
+
+* Learn about [resource management for Pods and Containers](docs/concepts/configuration/manage-resources-containers/).
```

Comment:
Oh, and:
```suggestion
* Learn about [resource management for Pods and Containers](/docs/concepts/configuration/manage-resources-containers/).
```

I missed this in my review. Would be good to fix that broken link before merge.

Reply:
good catch